core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -651,6 +657,17 @@ export class TopicEditorTabComponent implements OnInit, OnDestroy {
     );
     this.initEditor();
     this._initStorySummaries();
+    this.MAX_CHARS_IN_TOPIC_NAME = AppConstants.MAX_CHARS_IN_TOPIC_NAME;
```

Comment:
These values are used without {{ and  }}, see [here](https://github.com/oppia/oppia/pull/20081/files#diff-d73ca3bceeee4f9b56008923e0c7a206c800b6f307709a74a8a2f36cef75305dR79)

Do they work?

Reply:
Argh, yeah, sorry, the line is wrong.

Yeah, thanks for explaining, I'm not too good at html